TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/feature_selection/FeatureSelectionPersistence.java

```diff
@@ -0,0 +1,11 @@
+package com.novoda.tpbot.feature_selection;
+
+public interface FeatureSelectionPersistence {
```

Comment:
ATM the persistence is `SharedPreferences` but we should be able to swap it out.